### ISSUE TITLE
Windows/tabs to sharpen the feel of the game

### ISSUE THREAD

**Issue by Heathendel** [rockfireredmoon/iceee#12]
I was just considering this the other day and think its a good idea, and a good way to give the server a more professional feeling.  

The idea is to add some small tabs for certain things aren't currently working or were never created.

/commands are convenient but a bit sloppy as far as appearance.

Here are some of things I think need adding:

LOOT SYSTEM: This would be a tab for groups where the party would agree before starting how they will divide the loot and the party leader would then set that system.  Ones that come to mind are Need and Greed, Loot Master ( party leader can collect loot to be divided after) and perhaps a class restricted loot system (where if you can use the item you are unable to pick it up)

PET VENDOR:  The Cosmetic pets are a fun little feature, but the window that opens with the /pet command is a bit messy.  Would be cool if we could have Pet vendors in major Cities and towns where a preview key would display a small picture of the pet rather then briefly spawn it.

CRAFTING WINDOW: although the new crafting system will feature this anyways i figured i'd add it to this list just for the heck of it.  Crafting tab would display a more organized and detailed window to show you what plans you know how to make and when selected display lvl, materials, stats.  

MOUNT VENDOR: This is another one that i assume will be added when the content is actually finished but having Mount Vendors in each Major City again with preview pics of the mount would be a nice touch. 

AUCTION HOUSE:  This feature would be fairly low priority but I've heard grumblings for an Auction House ever since open beta in SPM, so an Auction house again in Major Cities and Towns would be a popular feature and again may encourage trade more, but will be fairly useless feature until player numbers start to rise nevertheless would be good to have even if its not used as much early on.

WEAPON RE-FASHIONER:  The feature to refashion weapons has been amongst the most popular as small as it is.  However i think either making Armor refashioners into Equipment Re-fashioners which could do both armor and weapons would be a good thing even though some players will dislike not being able to refashion anywhere there will be much less lost and destroyed items which means less heart ache for Emerald when people come complaining about their lost item. 

Those are some major ones ive thought of im sure there are more we could add but these would be commonly used ones.

**Comment by rockfireredmoon** [comment]
Weapon refashioning already done.
